securedrop/tests/test_2fa.py

```diff
@@ -119,4 +121,4 @@ def test_bad_token_fails_to_verify_on_new_user_two_factor_page(self):
     def tearDownClass(cls):
         # Reset the module variables that were changed to mocks so we don't
         # break other tests
-        reload(journalist)
+        reload_module(journalist)
```

Comment:
Hm, I didn't get a flake8 issue here. However for future Python 3 compatibility this is fine :+1: (`from imp import reload` would also work)